platform/core/src/services/HangingProtocolService/HPMatcher.js

```diff
@@ -36,7 +36,13 @@ const match = (metadataInstance, rules, customAttributeRetrievalCallbacks) => {
 
     // Create a single attribute object to be validated, since metadataInstance is an
     // instance of Metadata (StudyMetadata, SeriesMetadata or InstanceMetadata)
-    const attributeValue = metadataInstance[attribute];
+    let attributeValue = metadataInstance[attribute];
```

Comment:
The study level information is very sparse, because it doesn't actually start with a study query, so generate some of the study level information automatically.